dotcms-postman/src/main/resources/postman/ContentResourceV1.postman_collection.json

```diff
@@ -207,1437 +161,1330 @@
+											"    const response = pm.response.json();",
+											"    pm.expect(response.errors).to.be.an('array').that.is.empty;",
+											"});",
+											"",
+											"var jsonData = pm.response.json();",
+											"pm.collectionVariables.set(\"parentId1\", jsonData.entity.identifier);",
+											"pm.collectionVariables.set(\"parentInode1\", jsonData.entity.inode);",
+											"",
+											""
+										],
+										"type": "text/javascript",
+										"packages": {}
+									}
+								}
+							],
+							"request": {
+								"method": "PUT",
+								"header": [
+									{
+										"key": "Content-Type",
+										"value": "application/json"
+									}
+								],
+								"body": {
+									"mode": "raw",
+									"raw": "{\n    \"contentlet\": {\n        \"contentType\": \"ParentContentType\",\n        \"title\": \"parent 1\"\n    }\n}"
+								},
+								"url": {
+									"raw": "{{serverURL}}/api/v1/workflow/actions/default/fire/PUBLISH",
+									"host": [
+										"{{serverURL}}"
+									],
+									"path": [
+										"api",
+										"v1",
+										"workflow",
+										"actions",
+										"default",
+										"fire",
+										"PUBLISH"
+									]
+								},
+								"description": "Create the a parent content instance"
+							},
+							"response": []
+						},
 						{
-							"key": "password",
-							"value": "admin",
-							"type": "string"
+							"name": "Create parent2 Instance",
+							"event": [
+								{
+									"listen": "test",
+									"script": {
+										"exec": [
+											"pm.test(\"Validate that the 'errors' array is empty\", function () {",
+											"    const response = pm.response.json();",
+											"    pm.expect(response.errors).to.be.an('array').that.is.empty;",
+											"});",
+											"",
+											"var jsonData = pm.response.json();",
+											"pm.collectionVariables.set(\"parentId2\", jsonData.entity.identifier);",
+											"pm.collectionVariables.set(\"parentInode2\", jsonData.entity.inode);",
+											"",
+											""
+										],
+										"type": "text/javascript",
+										"packages": {}
+									}
+								}
+							],
+							"request": {
+								"method": "PUT",
+								"header": [
+									{
+										"key": "Content-Type",
+										"value": "application/json"
+									}
+								],
+								"body": {
+									"mode": "raw",
+									"raw": "{\n    \"contentlet\": {\n        \"contentType\": \"ParentContentType\",\n        \"title\": \"parent 2\"\n    }\n}"
+								},
+								"url": {
+									"raw": "{{serverURL}}/api/v1/workflow/actions/default/fire/PUBLISH",
+									"host": [
+										"{{serverURL}}"
+									],
+									"path": [
+										"api",
+										"v1",
+										"workflow",
+										"actions",
+										"default",
+										"fire",
+										"PUBLISH"
+									]
+								},
+								"description": "Create the a parent content instance"
```

Comment:
Create the **a** parent content instance

Reply:
done!